.github/workflows/main.yml

```diff
@@ -45,6 +45,15 @@ jobs:
             macos-latest
         ]
         node: [18.x, 20.x, latest]
+        weval-version:
+          - 'v0.3.2'
```

Comment:
My concern here is the same as on the other CI change - this is a maintenance burden now, separately to our ComponentizeJS Weval version.

I'm not sure how to square that?

Reply:
I can get the latest weval version dynamically -- multiple ways I can do that but I'll do it with `curl` (turns out if you try to hit `https://github.com/bytecodealliance/weval/releases/latest` you get redirected to the latest release!)

Reply:
Done! Weval version is now being downloaded dynamically